scripts/ci/readme_metadata_style_check/README_style_checker.py

```diff
@@ -0,0 +1,435 @@
+    trailing folder in the path; set to certain index to get other parts.
+    :return: The folder name.
+    """
+    return os.path.normpath(path).split(os.path.sep)[index]
+
+
+def parse_head(head_string: str) -> (str, str):
+    """
+    Parse the head of README and get title and description.
+    :param head_string: A string containing title, description and images.
+    :return: Stripped title and description strings.
+    """
+    # Split title section and rule out empty lines.
+    parts = list(filter(bool, head_string.splitlines()))
+    if len(parts) < 3:
+        raise Exception('README should contain title, description and image.')
+    title = parts[0].lstrip('# ').rstrip()
+    description = parts[1].strip()
+    return title, description
+
+
+def check_apis(apis_string: str) -> typing.Set[str]:
+    """
+    Check the format for `Relevant API` section.
+    :param apis_string: A multiline string containing all APIs.
+    :return: A set of APIs. Throws if format is wrong.
+    """
+    stripped = apis_string.strip()
+    apis = list(stripped.splitlines())
+    if not apis:
+        raise Exception('Empty Relevant APIs.')
+    s = set()
+    stripped_apis = []
+    for api in apis:
+        # Bullet is checked by the linter, no need to check here.
+        a = api.lstrip('*- ').rstrip()
+        s.add(a)
+        stripped_apis.append(a)
+        if '`' in a:
+            raise Exception('API should not include backticks.')
+    if '' in s:
+        raise Exception('Empty line in APIs.')
+    if len(apis) > len(s):
+        raise Exception('Duplicate APIs.')
+    if stripped_apis != sorted(stripped_apis, key=str.casefold):
+        raise Exception('APIs are not sorted.')
+    return s
+
+
+def check_tags(tags_string: str) -> typing.Set[str]:
+    """
+    Check the format for `Tags` section.
+    :param tags_string: A string containing all tags, with comma as delimiter.
+    :return: A set of tags. Throws if format is wrong.
+    """
+    tags = tags_string.split(',')
+    if not tags:
+        raise Exception('Empty tags.')
+    s = set()
+    stripped_tags = []
+    for tag in tags:
+        t = tag.strip()
+        s.add(t)
+        stripped_tags.append(t)
+        if t.lower() != t and t.upper() != t and t.capitalize() != t \
+                and t not in exception_proper_nouns:
+            raise Exception(f'Wrong letter case for tag: "{t}".')
+    if '' in s:
+        raise Exception('Empty char in tags.')
+    if ', '.join(stripped_tags) != tags_string.strip():
+        raise Exception('Extra whitespaces in tags.')
+    if len(tags) > len(s):
+        raise Exception('Duplicate tags.')
+    if stripped_tags != sorted(stripped_tags, key=str.casefold):
+        raise Exception('Tags are not sorted.')
+    return s
+
+
+def check_sentence_case(string: str) -> None:
+    """
+    Check if a sentence follows 'sentence case'. A few examples below.
+    Hello world! -> YES
+    I'm a good guy. -> YES
+    a man and a gun. -> NO
```

Comment:
minor comment but stood out for me...could we make these examples a bit more inclusive and less masculine/weaponised? :D some esri related suggestions below! 

```suggestion
    We're a mapping company. -> YES
    a map and a globe. -> NO
```

Reply:
Thanks I've skimmed the detail in this and agree these should be updated. Adding these changes now.